Modules/_sqlite/connection.c

```diff
@@ -118,6 +118,8 @@ int pysqlite_connection_init(pysqlite_Connection* self, PyObject* args, PyObject
         return -1;
     }
     Py_BEGIN_ALLOW_THREADS
+    /* No need to use sqlite3_open_v2 as sqlite3_open(filename, db) is the
+       same as sqlite3_open_v2(filename, db, SQLITE_OPEN_READWRITE | SQLITE_OPEN_CREATE, NULL)*/
```

Comment:
Nitpick: `FOO|BAR` syntax instead of `FOO | BAR`.

Also, finish the line with a full stop and space:

```c
/* [...] NULL). */
```